nipype/interfaces/freesurfer/model.py

```diff
@@ -401,6 +401,21 @@ class GLMFitInputSpec(FSTraitedSpec):
     synth = traits.Bool(argstr="--synth", desc="replace input with gaussian")
     resynth_test = traits.Int(argstr="--resynthtest %d", desc="test GLM by resynthsis")
     profile = traits.Int(argstr="--profile %d", desc="niters : test speed")
+    mrtm1 = InputMultiPath(
+        traits.Tuple(File(exists=True), File(exists=True)),
+        argstr="--mrtm1 %s %s...",
+        desc="RefTac TimeSec : perform MRTM1 kinetic modeling",
+    )
+    mrtm2 = InputMultiPath(
+        traits.Tuple(File(exists=True), File(exists=True), traits.Float),
+        argstr="--mrtm2 %s %s %f...",
+        desc="RefTac TimeSec k2prime : perform MRTM2 kinetic modeling",
+    )
+    logan = InputMultiPath(
+        traits.Tuple(File(exists=True), File(exists=True), traits.Float),
+        argstr="--logan %s %s %f...",
+        desc="RefTac TimeSec tstar   : perform Logan kinetic modeling",
+    )
```

Comment:
Just want to check in on these `InputMultiPath`s. To be clear, what `InputMultiPath(Tuple(trait1, trait2))` does is allow you some flexibility to pass a single instance to a tool that can take multiple:

| Input to interface | Received by interface |
|---|---|
| `(a, b)` | `[(a, b)]` |
| `[(a, b)]` | `[(a, b)]` |
| `[(a, b), (c, d)]` |`[(a, b), (c, d)]` |

From my reading, these things don't actually take multiple. `--mrtm1 file1 file2 --mrtm1 file3 file4` is not something the tool supports. I might be wrong though.

If these really can only take one set of inputs, then instead we want:

```suggestion
    mrtm1 = traits.Tuple(
        File(exists=True),
        File(exists=True),
        argstr="--mrtm1 %s %s",
        desc="RefTac TimeSec : perform MRTM1 kinetic modeling",
    )
    mrtm2 =  traits.Tuple(
        File(exists=True),
        File(exists=True),
        traits.Float,
        argstr="--mrtm2 %s %s %f",
        desc="RefTac TimeSec k2prime : perform MRTM2 kinetic modeling",
    )
    logan = traits.Tuple(
        File(exists=True),
        File(exists=True),
        traits.Float,
        argstr="--logan %s %s %f",
        desc="RefTac TimeSec tstar   : perform Logan kinetic modeling",
    )
```

Reply:
You are right - thanks for pointing this out!